### ISSUE TITLE
8.0.0 on hotInit error

### ISSUE THREAD

**Issue by cn-xufei** [handsontable/vue-handsontable-official#190]
### Steps to reproduce

1. 
```
<template>
  <div id="app">
    <hot-table
      :data="data"
      colHeaders="true"
      rowHeaders="true"
      width="600"
      height="300"
      licenseKey="non-commercial-and-evaluation"
    ></hot-table>
  </div>
</template>

<script>
import { HotTable } from '@handsontable/vue'

export default {
  name: 'App',
  components: { HotTable },
  data() {
    return {
      data: [
        ['', 'Tesla', 'Mercedes', 'Toyota', 'Volvo'],
        ['2019', 10, 11, 12, 13],
        ['2020', 20, 11, 14, 13],
        ['2021', 30, 15, 12, 13]
      ]
    }
  }
}
</script>

<style>
@import '~handsontable/dist/handsontable.full.css';
#app {
  font-family: Avenir, Helvetica, Arial, sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  text-align: center;
  color: #2c3e50;
  margin-top: 60px;
}
</style>

```

### Demo

https://jsfiddle.net/handsoncode/dezfbgjt/

### Your environment
* Vue wrapper version: 2.6.11
* Handsontable version: 8.0.0
* Browser Name and version: chrome 85
* Operating System: mac

It is possible under the development mode
Error accessing page after build

DOMException: Failed to execute 'insertBefore' on 'Node': This node type does not support this method.

![image](https://user-images.githubusercontent.com/14144811/91660425-5665cf80-eb08-11ea-9650-620896e81d3f.png)

**Comment by AMBudnik** [comment]
Hi @cn-xufei 

Here's an updated demo https://codesandbox.io/s/advanced-vue-hot-column-implementation-722-411-forked-sjf4w?file=/src/App.vue

Let me know if you need anything else.

**Comment by cn-xufei** [comment]
> Hi @cn-xufei
> 
> Here's an updated demo https://codesandbox.io/s/advanced-vue-hot-column-implementation-722-411-forked-sjf4w?file=/src/App.vue
> 
> Let me know if you need anything else.

Thanks. Demo no problem.
But my problem is that on version 8.0, errors are reported after the project is packaged.
```
DOMException: Failed to execute 'insertBefore' on 'Node': This node type does not support this method.
```
My environment
```json
{
    "vue": "^2.6.11",
    "@handsontable/vue": "^5.0.1",
    "handsontable": "^8.0.0"
}
```

**Comment by Freshstrong** [comment]
I also encountered the same problem. Like the @ CN Xufei environment, I also reported this error

**Comment by AMBudnik** [comment]
@cn-xufei by 'packed' do you mean the webpack's `minimize` option?

If so it has been reported here https://github.com/handsontable/vue-handsontable-official/issues/188

**Comment by Freshstrong** [comment]
I didn't find "minimize" in my project
![image](https://user-images.githubusercontent.com/23231852/91831563-ce520800-ec76-11ea-9439-dcc2a714d2c2.png)

**Comment by AMBudnik** [comment]
Can I ask you @cn-xufei and @Freshstrong to send me any demo where this issue can be replicable? It seems that we won't be able to replicate the same using Sandbox.

**Comment by Freshstrong** [comment]
I have solved the problem of webpack, minimize option. I added "minimize" to false and repackaged it to solve this problem

![image](https://user-images.githubusercontent.com/23231852/91833111-d9a63300-ec78-11ea-9363-b55b522567fb.png)

**Comment by AMBudnik** [comment]
I'm glad to hear that @Freshstrong Thank you for the update.
@cn-xufei does it solve your problem as well?

**Comment by cn-xufei** [comment]
> @cn-xufei by 'packed' do you mean the webpack's `minimize` option?
> 
> If so it has been reported here #188

Thanks.
Yes, that's the question

**Comment by cn-xufei** [comment]
> I have solved the problem of webpack, minimize option. I added "minimize" to false and repackaged it to solve this problem
> 
> ![image](https://user-images.githubusercontent.com/23231852/91833111-d9a63300-ec78-11ea-9363-b55b522567fb.png)

Thanks.
This can only be done temporarily. Closing this configuration will affect code compression

**Comment by Freshstrong** [comment]
> > I have solved the problem of webpack, minimize option. I added "minimize" to false and repackaged it to solve this problem
> > ![image](https://user-images.githubusercontent.com/23231852/91833111-d9a63300-ec78-11ea-9363-b55b522567fb.png)
> 
> Thanks.
> This can only be done temporarily. Closing this configuration will affect code compression

At present, there is no better way. I hope the author can solve this problem in the next version

**Comment by AMBudnik** [comment]
Yes, that is only a workaround.

@cn-xufei @Freshstrong I will update you as soon as we get that issue fixed.

**Comment by wojciechczerniak** [comment]
Duplicate of #188

**Comment by AMBudnik** [comment]
https://github.com/handsontable/vue-handsontable-official/issues/188 has been solved in v 5.1.0
